stan/inc2prev.stan

```diff
@@ -10,17 +10,20 @@ functions {
 data {
   int ut; // initial period (before data starts)
   int t; // number of time points to model
+  int n; // number of time series to model
+  int n_ab; // number of antibody time series to model
+  int ab_index[n_ab]; // antibody dynamics indices
   int obs; // number of prevalence observations
-  int ab_obs; // number of antibody prevalence observations
-  vector[obs] prev; // observed positivity prevalence
-  vector[obs] prev_sd2; // squared standard deviation of observed positivity prevalence
-  vector[ab_obs] ab; // observed antibody posivitiy prevalence
-  vector[ab_obs] ab_sd2; // squared standard deviation of observed antibody prevalence
+  int ab_obs; // presence/absence of antibody observations (1 = present; 0 = absent)
+  matrix[n, obs] prev; // observed positivity prevalence
```

Comment:
This might require some less painful code further down if it was stored and passed as an array of vectors